packages/asc-ui/src/components/Accordion/AccordionStyle.ts

```diff
@@ -0,0 +1,60 @@
+import styled from '@datapunt/asc-core'
+import Button from '../Button'
+import { IconStyle } from '../Icon'
+import { DEFAULT_BUTTON_HEIGHT } from '../Button/ButtonStyle'
+import {
+  DEFAULT_OUTLINE_WIDTH,
+  themeColor,
+  themeSpacing,
+} from '../../utils/themeUtils'
+
+export type Props = {
+  open?: boolean
+}
+
+const AccordionContent = styled.div`
+  transition: border-color 0.1s ease-in-out;
+  border: 2px solid ${themeColor('tint', 'level3')};
+  border-top: none;
+  padding: ${themeSpacing(4, 5)};
+`
+
+const AccordionButtonContent = styled.span`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
Is it wcag proof to use `text-overflow: ellipsis;`? https://www.w3.org/TR/WCAG20-TECHS/G179.html

Reply:
I will set a title attribute, that'll solve it imho

Reply:
since i already have a title attribute on the button I'll leave this, ok?